Example_SwiftUI/Example_SwiftUI/Example_SwiftUIApp.swift

```diff
@@ -41,3 +41,30 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         return [viewController]
     }
 }
+
+// MARK: - Alamofire bugfix in uploadProgress
+
+extension AppDelegate: CustomHTTPProtocolDelegate {
+    func urlSession(
+        _ protocol: URLProtocol,
+        _ session: URLSession,
+        task: URLSessionTask,
+        didSendBodyData bytesSent: Int64,
+        totalBytesSent: Int64,
+        totalBytesExpectedToSend: Int64
+    ) {
+
+        Session.default.session.getAllTasks { tasks in
+            let uploadTask = tasks.first(where: { $0.taskIdentifier == task.taskIdentifier }) ?? task
+            Session.default.rootQueue.async {
+                Session.default.delegate.urlSession(
+                    session,
+                    task: uploadTask,
+                    didSendBodyData: bytesSent,
+                    totalBytesSent: totalBytesSent,
+                    totalBytesExpectedToSend: totalBytesExpectedToSend
+                )
+            }
+        }
+    }
+}
```

Comment:
Can this become part of the default extension of `CustomHTTPProtocolDelegate`. I believe it would work as expected in a blank project (without Alamofire) too.

Reply:
if we use #if import(Alamofire), @heitara could you make the modifications?